src/torchpme/tuning/tuner.py

```diff
@@ -0,0 +1,325 @@
+import math
+import time
+from typing import Optional
+
+import torch
+import vesin.torch
+
+from ..calculators import Calculator
+from ..potentials import CoulombPotential
+from ..utils import _validate_parameters
+
+
+class TuningErrorBounds(torch.nn.Module):
+    """
+    Base class for error bounds. This class calculates the real space error and the
+    Fourier space error based on the error formula. This class is used in the tuning
+    process. It can also be used with the :class:`torchpme.tuning.tuner.TunerBase` to
+    build up a custom parameter tuner.
+
+    :param charges: atomic charges
+    :param cell: single tensor of shape (3, 3), describing the bounding
+    :param positions: single tensor of shape (``len(charges), 3``) containing the
+        Cartesian positions of all point charges in the system.
+    """
+
+    def __init__(
+        self,
+        charges: torch.Tensor,
+        cell: torch.Tensor,
+        positions: torch.Tensor,
+    ):
+        super().__init__()
+        self._charges = charges
+        self._cell = cell
+        self._positions = positions
+
+    def forward(self, *args, **kwargs):
+        return self.error(*args, **kwargs)
```

Comment:
What does this mean? Schouldn't there be a not implemented error raised? As we do it for the `calculators` and `potentials`?